Library/Homebrew/cmd/missing.rb

```diff
@@ -6,15 +6,40 @@ module Homebrew
   def missing
     return unless HOMEBREW_CELLAR.exist?
 
-    ff = if ARGV.named.empty?
+    formulae = if ARGV.named.empty?
       Formula.installed
     else
       ARGV.resolved_formulae
     end
+    if ARGV.json == "v1"
+      missing = print_missing_json(formulae)
+    else
+      missing = print_missing(formulae)
+    end
+    Homebrew.failed = ARGV.resolved_formulae.any? && missing.any?
```

Comment:
Oh, OK, I was thinking I'd just rename this one, but if you'd like them separated that's fine.